tiatoolbox/tools/registration/wsi_registration.py

```diff
@@ -1,5 +1,119 @@
+import cv2
 import numpy as np
+import scipy.ndimage as ndi
 from skimage import exposure, filters, morphology
+from skimage.util import img_as_float
+
+from tiatoolbox.utils.metrics import dice
+
+
+def _check_dimen(fixed_img, moving_img, fixed_mask, moving_mask):
```

Comment:
Suggest calling thins `_chem_dims` (numpy and torch use dim as short for dimension) or `_check_dimensions`, I think `dimen` is a bit more obscure / unclear for a name.

Reply:
https://github.com/TissueImageAnalytics/tiatoolbox/blob/ee13a1f6f90b392278ac14b6c28d7d8465c083b7/tiatoolbox/tools/registration/wsi_registration.py#L12